playwright/test/workPrototype.test.ts

```diff
@@ -0,0 +1,81 @@
+import {
+  workWithPhysicalAndDigitalLocation,
+  workWithRequestablePhysicalItem,
+  workWithPhysicalLocationOnly,
+  workWithDigitalLocationOnly,
+} from './contexts';
+import { makeDefaultToggleAndTestCookies } from './helpers/utils';
+import { baseUrl } from './helpers/urls';
+
+const domain = new URL(baseUrl).host;
+
+// Turn on the showPhysicalItems toggle
+beforeAll(async () => {
+  const defaultToggleAndTestCookies = await makeDefaultToggleAndTestCookies(
+    domain
+  );
+  const toggleOverrides = [
+    {
+      name: 'toggle_showPhysicalItems',
+      value: 'true',
+    },
+  ];
+  const overriddenCookies = defaultToggleAndTestCookies.map(cookie => {
+    const matchingOverrideCookie = toggleOverrides.find(
+      override => override.name === cookie.name
+    );
+    if (matchingOverrideCookie) {
+      return {
+        ...cookie,
+        value: matchingOverrideCookie.value,
+      };
+    } else {
+      return cookie;
+    }
+  });
+  await context.addCookies([
+    { name: 'WC_cookiesAccepted', value: 'true', domain: domain, path: '/' },
+    { name: 'WC_globalAlert', value: 'true', domain: domain, path: '/' },
+    ...overriddenCookies,
+  ]);
+});
+
+async function getWhereToFindItAndEncoreLink() {
+  const whereToFindIt = await page.$('h2:has-text("Where to find it")');
+  const encoreLink = await page.$('a:has-text("Request item")');
+
+  return {
+    whereToFindIt,
+    encoreLink,
+  };
+}
+
+describe(`Scenario 1: a user wants to see relevant information about where a work's items are located and the access status of those items where applicable`, () => {
+  test(`works with a physical location item only display a 'where to find it' section with a link`, async () => {
+    await workWithPhysicalLocationOnly();
+    const { whereToFindIt, encoreLink } = await getWhereToFindItAndEncoreLink();
+    expect(whereToFindIt).toBeTruthy();
+    expect(encoreLink).toBeTruthy();
+  });
+
+  test(`works with a physical location item available to request display an access status`, async () => {
+    await workWithRequestablePhysicalItem();
+    const status = await page.waitForSelector('th:has-text("Status")');
+    expect(status).toBeTruthy();
+  });
+
+  // Not sure this is actually what we want
```

Comment:
Could you explain this comment further? What is it about this that is questionable?

Reply:
I thought we wanted to prioritise people using the digital version but not prevent them from requesting the physical one (will check)

Reply:
Ah, great, could just this comment in here.